plugin-hrm-form/src/permissions/index.ts

```
@@ -18,30 +18,28 @@ export const PermissionActions = {
 
 type PermissionActionsKeys = keyof typeof PermissionActions;
 export type PermissionActionType = typeof PermissionActions[PermissionActionsKeys];
-type Version = 'v1' | 'za-v1';
+type PermissionConfig = 'zm' | 'za';
 type Rule = (isSupervisor: boolean, isCreator: boolean, isCaseOpen: boolean) => boolean;
 type Rules = {
   canEditCaseSummary: Rule;
   canEditGenericField: Rule;
   canReopenCase: Rule;
 };
 
-const rulesMap: { [version in Version]: Rules } = {
-  v1: v1Rules,
-  'za-v1': zaV1Rules,
+const rulesMap: { [permissionConfig in PermissionConfig]: Rules } = {
+  zm: zmRules,
+  za: zaRules,
 };
 
-export const getPermissionsForCase = (
-  version: t.Case['info']['definitionVersion'],
-  twilioWorkerId: t.Case['twilioWorkerId'],
-  status: t.Case['status'],
-) => {
-  if (!version || !twilioWorkerId || !status) return { can: undefined };
+const fallbackRules = zaRules;
+
+export const getPermissionsForCase = (twilioWorkerId: t.Case['twilioWorkerId'], status: t.Case['status']) => {
+  const { workerSid, isSupervisor, permissionConfig } = getConfig();
+  if (!permissionConfig || !twilioWorkerId || !status) return { can: undefined };
 
-  const { workerSid, isSupervisor } = getConfig();
   const isCreator = workerSid === twilioWorkerId;
   const isCaseOpen = status === 'open';
-  const rules = rulesMap[version];
+  const rules = rulesMap[permissionConfig] || fallbackRules;
```

Should we silently continue here using ZA rules? Maybe a console.error at least to raise the issue?

---

Good idea. I'll keep the fallback so it won't ever crash, but call `console.error` so we can track it.